runtime/include/gpu/cuda/chpl-gpu-gen-includes.h

```diff
@@ -26,6 +26,7 @@
 #include "chpltypes.h"
 #include "chpl-comm.h"
 
+extern __device__ c_nodeid_t chpl_nodeID;
```

Comment:
I imagine over time we're going to get a collection of these global variables. Just seeing this code it wouldn't be obvious to me that the actual would be something being generated by our compiler. You might want to add a comment explaining that the actual instance of this variable is generated by the chpl compiler into our binary for the GPU and explain why we do it that way rather than just declaring the `__device__` variable in the runtime outright.